modules/opengl/include/modules/opengl/shader/shader.h

```diff
@@ -120,6 +120,7 @@ class IVW_MODULE_OPENGL_API Shader {
     void link();
     void build();
     bool isReady() const;  // returns whether the shader has been built and linked successfully
+    bool checkLinkStatus() const;
```

Comment:
Do we have to introduce an other "ready" function in the public interface.

Reply:
I will make this one private